src/mytnt.c

```diff
@@ -145,8 +145,9 @@ int mytnt_stmt_execute(MYTNT_STMT *stmt) {
 	}
 
 	if (reply->error) {
-		stmt->mytnt->error_info = (char *) malloc(1024 * sizeof(char));
-		strcpy(stmt->mytnt->error_info, reply->error);
+		int error_len = reply->error_end - reply->error;
+		stmt->mytnt->error_info = (char *)calloc(error_len + 1, sizeof(char));
+		strncpy(stmt->mytnt->error_info, reply->error, error_len);
```

Comment:
I guess that the problem is that there is an error message from tarantool in the given workload that doesn't fit into 1024 bytes. I would like to see a brief description of the problem that the patch aims to solve. If my guess it correct, just an example of such error message should be enough.

If it solves a theoretical buffer overflow, it is OK too. But, please, describe the reason of the change in the commit message anyway.

Reply:
The problem was that reply->error is not null-terminated but it was used as a source string in strcpy.

Reply:
OK, got it. Can you add the problem description in the commit message?

Reply:
Done.

Reply:
Thanks!